js/modals.js

```diff
@@ -0,0 +1,85 @@
+
+
+const closePopUp = document.getElementsByClassName('popup_close');
+
+const openPopUp_1 = document.getElementById('popup1_open');
+const openPopUp_2 = document.getElementById('popup2_open');
+const openPopUp_3 = document.getElementsByClassName('popup3_open');
+
+const open_menu = document.getElementById('open_menu')
+
+const popUp1 = document.getElementById('popUp1')
+const popUp2 = document.getElementById('popUp2')
+const popUp3 = document.getElementById('popUp-form')
+const popUp = document.getElementById('popUp')
+
+const menu = document.getElementById('menu')
+
+let countWindows = 0
+
+for (var i = 0; i < closePopUp.length; i++) {
```

Comment:
почему var 
исправить на акутальный стандарт es6+
```suggestion
for (const popUp of popUps) { 
```
тк будет массив
или
```suggestion
popUps.forEach((popUp) => {
     //....
})
```